include/votca/csg/pdbwriter.h

```diff
@@ -22,12 +22,15 @@
 #include <votca/csg/topology.h>
 #include <votca/csg/trajectorywriter.h>
 #include <votca/tools/constants.h>
+#include <votca/tools/unitconverter.h>
 
 namespace votca {
 namespace csg {
 
 class PDBWriter : public TrajectoryWriter {
  public:
+  const tools::DistanceUnit distance_unit = tools::DistanceUnit::angstroms;
```

Comment:
Why `tools::` and not `csg::`?

Reply:
All of the types come from the unitconverter header in tools.